protocols/syslog/syslog_net.c

```diff
@@ -39,7 +39,8 @@ syslog_net_init(void)
 
   syslog_conn = uip_udp_new(&ip, HTONS(SYSLOG_PORT), syslog_net_main);
 
-  if(! syslog_conn) {
+  if (!syslog_conn)
+  {
     debug_printf("syslog: couldn't create connection\n");
```

Comment:
We are both right. The debug output should be enclosed in #ifndef DEBUG_USE_SYSLOG.